tests/python/frontend/pytorch/test_forward.py

```diff
@@ -1428,6 +1428,48 @@ def test_forward_upsample3d():
     verify_model(torch.nn.Upsample(scale_factor=2, mode='trilinear', align_corners=True).eval(), inp)
 
 
+def test_forward_nms():
+    """dynamic Non-Maximum Suppression"""
+    torch.set_grad_enabled(False)
+    class NonMaxSupression1(Module):
+        def forward(self, *args):
+            return torchvision.ops.nms(args[0], args[1], 0.3)
+
+    class NonMaxSupression2(Module):
+        def forward(self, *args):
+            from torchvision.ops import nms
+            return torchvision.ops.nms(args[0], args[1], 0.5)
+
+    class NonMaxSupression3(Module):
+        def forward(self, *args):
+            from torchvision.ops import nms
+            return torchvision.ops.nms(args[0], args[1], 0.9)
+
```

Comment:
Only a single class should be enough. The third argument can be initialized in the constructor with different values.